sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -52,326 +51,383 @@ private Transforms() {
-     * {@link com.azure.ai.formrecognizer.implementation.models.FieldValue#getValueDate()}.
+     * @param words A list of word reference elements returned by the service.
+     * @param pageNumber The 1 based page number on which this word element exists.
      *
-     * @return The {@link StringValue}.
+     * @return The list of {@code FormWord words}.
      */
-    private static DateValue toFieldValueDate(com.azure.ai.formrecognizer.implementation.models.FieldValue
-                                                      serviceDateValue) {
-        return new DateValue(serviceDateValue.getText(), toBoundingBox(serviceDateValue.getBoundingBox()),
-            serviceDateValue.getValueDate(), serviceDateValue.getPage());
+    static IterableStream<FormWord> toWords(List<TextWord> words, Integer pageNumber) {
+        List<FormWord> extractedWordList = words.stream()
+            .map(textWord -> new FormWord(textWord.getText(), toBoundingBox(textWord.getBoundingBox()), pageNumber,
+                textWord.getConfidence())).collect(Collectors.toList());
+        return new IterableStream<FormWord>(extractedWordList);
     }
 
     /**
-     * Helper method to convert the service returned {@link com.azure.ai.formrecognizer.implementation.models.FieldValue#getValueTime()}
-     * to a SDK level {@link TimeValue}.
-     *
-     * @param serviceDateValue The string value returned by the service in
-     * {@link com.azure.ai.formrecognizer.implementation.models.FieldValue#getValueTime()} .
+     * Helper method to convert the service level modeled eight numbers representing the four points to SDK level
+     * {@link BoundingBox}.
      *
-     * @return The {@link TimeValue}.
+     * @param serviceBoundingBox A list of eight numbers representing the four points of a box.
      *
+     * @return A {@link BoundingBox}.
      */
-    private static TimeValue toFieldValueTime(com.azure.ai.formrecognizer.implementation.models.FieldValue
-                                                  serviceDateValue) {
-        return new TimeValue(serviceDateValue.getText(), toBoundingBox(serviceDateValue.getBoundingBox()),
-            serviceDateValue.getValueTime(), serviceDateValue.getPage());
-        // TODO: currently returning a string, waiting on swagger update.
+    private static BoundingBox toBoundingBox(List<Float> serviceBoundingBox) {
+        if (CoreUtils.isNullOrEmpty(serviceBoundingBox)) {
+            return null;
+        }
+        Point topLeft = new Point(serviceBoundingBox.get(0), serviceBoundingBox.get(1));
+        Point topRight = new Point(serviceBoundingBox.get(2), serviceBoundingBox.get(3));
+        Point bottomLeft = new Point(serviceBoundingBox.get(4), serviceBoundingBox.get(5));
+        Point bottomRight = new Point(serviceBoundingBox.get(6), serviceBoundingBox.get(7));
+        return new BoundingBox(Arrays.asList(topLeft, topRight, bottomLeft, bottomRight));
```

Comment:
It was originally designed to be accepting specific four parameters in the constructor but with service feedback that it could be in future grow to accept more than just 8 points extended, updated this to accept a list.